webcompat/templates/issue-wizard-form.html

```diff
@@ -230,32 +230,31 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
     </div>
   </div>
 
-  <div class="step-container step-description col">
+  <div class="step-container step-description col" data-min-char="false">
```

Comment:
Cool, so we can toggle it on or off based on this.

I assumed we were going to do some A/B testing based on the existing AB test stuff we have, this approach would be compatible with that -- you set the value in the template based on active experiment.